game.sy

```diff
@@ -99,8 +99,75 @@ player_new :: fn pos: (float, float) -> Player {
 
 :Player is :Entity
 
+Puck :: blob {
+    pos: (float, float),
+    rot: float,
+    vel: float,
+    acc: float,
```

Comment:
Is this how we want to represent movement? Feels like we want a lot of forces acting on this - and tuple addition is defined in the language.